.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+name: Bug Report
+description: Identify bugs in of our Upbound's products.
+labels: bug,needs:triage
+title: "<component name>: <bug overview>"
+body:
```

Comment:
suggestion(non-blocking): Should we have a dropdown field with a list of Upbound products to report on? If so we can also remove <product name> from the issue title.